doc/examples/flare/lib/main.dart

```diff
@@ -0,0 +1,124 @@
+import 'dart:ui';
+import 'package:flutter/gestures.dart';
+import 'package:flame/flame.dart';
+import 'package:flame/game.dart';
+import 'package:flame/flare_animation.dart';
+import 'package:flame/text_config.dart';
+import 'package:flame/position.dart';
+
+import 'package:flutter/material.dart';
+
+void main() {
+  final game = MyGame();
+  runApp(game.widget);
+
+  Flame.util.addGestureRecognizer(TapGestureRecognizer()
+    ..onTapDown = (TapDownDetails evt) {
+      game.cycleAnimation();
+    });
+}
+
+class MyGame extends BaseGame {
+  final TextConfig fpsTextConfig = TextConfig(color: const Color(0xFFFFFFFF));
+
+  final paint = Paint()..color = Color(0xFFe5e5e5e5);
+  List<String> _animations = ["Stand", "Wave", "Jump", "Dance"];
+  int _currentAnimation = 0;
+
+  FlareAnimation flareAnimation;
+  FlareAnimation flareAnimation2;
+  FlareAnimation flareAnimation3;
+  FlareAnimation flareAnimation4;
```

Comment:
since you are using basegame, why don't you `add` those to the game? then you don't the references and can call super's update and render.